domain/lib/src/repository/sharedspacedocument/shared_space_document_repository.dart

```diff
@@ -42,4 +42,7 @@ abstract class SharedSpaceDocumentRepository {
       Uri baseUrl,
       SharedSpaceId sharedSpaceId,
       {WorkGroupNodeId parentNodeId});
+
+  Future<List<WorkGroupNode>> getAllChildNodes(
+    SharedSpaceId sharedSpaceId, [WorkGroupNodeId parentNodeId]);
```

Comment:
please reformat code

Reply:
In our project, what convention we are prefering? Because it exceed max code in one line :( Indeed, I prefer use trailing command like this https://flutter.dev/docs/development/tools/formatting#using-trailing-commas '_'

Reply:
I have never seen your convention before :(. I prefer, each parameter in each line in case exceed characters in line.

Reply:
And, we follow `pedantic` convention. Please check more details here: https://pub.dev/packages/pedantic

Reply:
It still there

Reply:
pls use `{}`